components/KiboShipping.vue

```diff
@@ -6,13 +6,44 @@
       class="sf-heading--left sf-heading--no-underline title"
     />
     <div class="user-addresses">
-      <UserSavedAddresses
-        :countries="countries"
-        :default-address="shippingAddress"
-        :addresses="userShippingAddresses"
-        :is-readonly="isReadonly"
-        @onSave="saveShippingAddress"
-        @validateForm="validateShippingDetails"
+      <SfLoader :loading="loadingSavedUserAddress">
+        <UserSavedAddresses
+          :countries="countries"
+          :default-address="shippingAddress"
+          :addresses="userShippingAddresses"
+          :is-readonly="isReadonly"
+          @onSave="saveShippingAddress"
+          @validateForm="validateShippingDetails"
+          @toggleAddressFormVisibility="toggleAddressFormVisibility"
+        />
+      </SfLoader>
+    </div>
+
+    <div v-if="isAddressFormOpen && isUserLoggedIn">
+      <SfCheckbox
+        name="shipping"
+        label="Save shipping address"
```

Comment:
I think internationalization should be used here.

Reply:
Resolved